src/Http/Response.php

```diff
@@ -2099,13 +2099,25 @@ public function checkNotModified(ServerRequest $request)
      *
      * @return string
      */
-    public function __toString()
+    public function getStringBody()
```

Comment:
Any chance of getting tests for new public methods? Adding public methods is something we've been trying not to do in bugfixes as well.

Reply:
Sure, but since __toString() uses it it is already covered by design right now.